client/src/controllers/scores/score_controller.js

```diff
@@ -90,13 +90,24 @@ class ScoreController {
 		}
 
 		this.Scores.update(this.data._id, updateData)
-		.then(() => {
-			self.$scope.$emit('reload')
-		}).catch(() => {
-			self.Notifications.error('Unable to update score: Possible network error.')
-		})
+			.then(() => {
+				self.$scope.$emit('reload')
+			}).catch(() => {
+				self.Notifications.error('Unable to update score: Possible network error.')
+			})
+	}
+
+	teamRoundError() {
+		let self = this
+
+		return !this._loading && this.data.round == null
 	}
 
+	teamNumberError() {

```

Comment:
Please make sure the names of these function completely reflect their meaning. This function's name isvery good, but the other function has nothing to do with team, so it could be `roundError` instead of `teamRoundError`